buildpipeline/Core-Setup-Linux-BT.json

```diff
@@ -624,6 +624,206 @@
         "failOnStandardError": "false"
       }
     },
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Package Runtime Dep - Fedora 26",
+      "timeoutInMinutes": 0,
+      "refName": "Task43",
+      "task": {
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "docker",
+        "arguments": "run --rm $(DockerCommonRunArgs_Rhel7) $(PB_GitDirectory)/Tools/msbuild.sh $(PB_GitDirectory)/src/pkg/packaging/dir.proj $(AdditionalMSBuildPackagingArguments) $(DistroSpecificMSBuildArguments) /p:OutputRid=fedora.26-$(PB_TargetArchitecture)",
```

Comment:
What do you think about factoring out the commonality in all of the package steps into a variable?  It would help ensure all package steps are doing the same thing.  Looks like the `OutputRid` may be the only difference.  It looks like a similar thing could be done for the publish steps as well.